core/assets/bundles/bundle_id_ID.properties

```diff
@@ -2362,10 +2377,10 @@ unit.risso.description = Menembak rentetan misil dan peluru kepada semua musuh t
 unit.minke.description = Menembak cangkang pembakar dan peluru standar kepada musuh darat terdekat.
 unit.bryde.description = Menembak artileri jarak jauh dan misil kepada musuh.
 unit.sei.description = Menembak rentetan misil dan peluru yang dapat menembus pelindung kepada musuh.
-unit.omura.description = Menembak railgun jarak jauh kepada musuh. Dapat memproduksi unit flare.
-unit.alpha.description = Melindungi Inti Bagian dari musuh. Dapat membangun struktur.
-unit.beta.description = Melindungi Inti Fondasi dari musuh. Dapat membangun struktur.
-unit.gamma.description = Melindungi Inti Nukleus dari musuh. Dapat membangun struktur.
+unit.omura.description = Menembak ledakan railgun jarak jauh kepada musuh. Dapat memproduksi unit flare.
```

Comment:
I don't recommend use *paser* in order to precise the translation. but no one didn't know what *paser* is?
just use *proyektil*/*peluru*, 
it similar as crossbow for sure.